pkg/mimirpb/compat.go

```diff
@@ -195,6 +238,106 @@ func fromSpansProtoToSpans(s []BucketSpan) []histogram.Span {
 	return spans
 }
 
+func FromHistogramToHistogramProto(timestamp int64, h *histogram.Histogram) Histogram {
+	return Histogram{
+		Count:          &Histogram_CountInt{CountInt: h.Count},
+		Sum:            h.Sum,
+		Schema:         h.Schema,
+		ZeroThreshold:  h.ZeroThreshold,
+		ZeroCount:      &Histogram_ZeroCountInt{ZeroCountInt: h.ZeroCount},
+		NegativeSpans:  fromSpansToSpansProto(h.NegativeSpans),
+		NegativeDeltas: h.NegativeBuckets,
```

Comment:
Should be `NegativeDeltas` or `NegativeCounts`?

_Same comment applies to `PositiveDeltas`._

Reply:
The comment in  the proto definition explains it: https://github.com/grafana/mimir/blob/5d158054ca1bdddc953e42ef80ef61589f8b4795/pkg/mimirpb/mimir.proto#L119
(aka https://github.com/prometheus/prometheus/blob/36b04efc7bce25f73cd3df8d5904f432920946c0/prompb/types.proto#L92)

Reply:
added comment